internal/builders/maven/README.md

```diff
@@ -81,11 +81,10 @@ env:
 jobs:
   build:
     permissions:
```

Comment:
the publisher call still uses write permissions:
```yaml
permissions:
      contents: write
      id-token: write
      actions: read
      packages: write
```, 

None of them are needed, correct?

Reply:
> None of them are needed, correct?

What do you mean here? IIUC we need these permissions after `packages: write` is removed from the call to the lowperms delegator.

```yaml
permissions:
  id-token: write
  contents: read
  actions: read
```

Reply:
sorry, that was meat as a comment for the publish, not the build. The publish should not need permissions